.travis.yml

```diff
@@ -1,26 +1,47 @@
+sudo: false
+
 language: python
 
 python:
   - 2.7
   - 3.3
   - 3.4
+  - 3.5
 
 env:
-  - DJANGO="Django>=1.5,<1.6"
-  - DJANGO="Django>=1.6,<1.7"
-  - DJANGO="Django>=1.7,<1.8"
-  - DJANGO="Django>=1.8,<1.9"
+  - DJANGO=1.6
```

Comment:
1.5 was annoying due to tests import but 1.6 is fine as far as I can see. Let's wait if there are no coding complications with 1.6